cmake/modules/RootCPack.cmake

```diff
@@ -132,6 +132,7 @@ endif()
 # We want to defer markdown to html conversion; CPACK_INSTALL_SCRIPTS allows this
 # but only takes a .cmake file as argument
 #
+configure_file(${CMAKE_SOURCE_DIR}/cmake/modules/CPackREADME.cmake.in ${CMAKE_SOURCE_DIR}/cmake/modules/CPackREADME.cmake @ONLY)
```

Comment:
See above: can you generate this file in the build directory? The source dir should not be touched.

Reply:
Should be fixed now.